app/views/admin_shelters/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<% @shelters.each do |shelter| %>
+  <div id="shelter-<%= shelter.id %>">
+    <h3><%= shelter.name %></h3>
+    <p>Created at: <%= shelter.created_at %></p>
+    <%= link_to "Update #{shelter.name}", "/shelters/#{shelter.id}/edit" %>
+    <%= link_to "Delete #{shelter.name}", "/shelters/#{shelter.id}", method: :delete %>
+  </div>
+<% end %>
+
+<div id="shelterPendingApps">
+  <h3>Shelters with pending applications</h3>
+  <% @shelters_with_apps.each do |shelter| %>
```

Comment:
Here is where I would just reach through your `@shelters` instance variable in your controller and call `.show_pending` method. 
```ruby
@shelters.show_pending.each do
```